oeps/processes/oep-0055-proc-project-maintainers.rst

```diff
@@ -0,0 +1,422 @@
+===========================
+OEP-55: Project Maintainers
+===========================
+.. list-table::
+   :widths: 25 75
+
+   * - OEP
+     - :doc:`OEP-0055 <oep-0055-proc-project-maintainers>`
+   * - Title
+     - Project Maintainers
+   * - Last Modified
+     - 2022-01-28
+   * - Authors
+     - Edward Zarecor <e0d@tcril.org>, Sarina Canelake <sarina@tcril.org>
+   * - Arbiter
+     - Jeremy Bowman <jbowman@2u.com>
+   * - Status
+     - Draft
+   * - Type
+     - Process
+   * - Created
+     - 2022-01-28
+   * - Review Period
+     - <start - target end dates for review>
+   * - Resolution
+     - <links to any discussions where the final status was decided>
+   * - References
+     - <links to any other relevant discussions or relevant related materials>
+
+Abstract
+========
+
+The Open edX platform is a service oriented platform for authoring and
+delivering online learning based on the `Django framework`_.  At the
+highest level, the platform is composed of numerous services (IDAs
+[#]_), micro-front ends (MFEs [#]_) and libraries.  The code for these
+services and libraries is managed in git and hosted on GitHub
+platform.
+
+Each component part of the Open edX platform requires maintenance to
+ensure that it meets, and continues to meet, the project definition of
+high quality.  That definition will include project standards,
+appropriate documentation, and fitness for use, among other things.
+That definition will evolve as project needs do and we learn what
+works and what doesn't.
+
+Component maintenance is work.  To ensure that that work is done, a
+person or persons must be recognized as responsible for doing it,
+and held accountable for its performance.
+
+The Open edX project, like many projects, calls this role
+"maintainer."  The notion of a project maintainer is typical of
+open-source projects and widely considered an best practice [#]_.
+
+The maintainer role is a leadership role that encompasses both
+technical and community management elements.  Both dimensions of the role are
+equally important.
+
+All, non-deprecated, components of the Open edX project must have one
+or more named maintainers.  When no such maintainers exists, this will
+be considered an urgent problem to solve.
+
+This OEP proposes a formal definition of the role of maintainer that
+will open to all qualified Open edX community members.  Maintainers
+will be responsible for the health of platform components, but not for
+their performance in any particular community member's production
+environment[#]_.
```

Comment:
Nit: I think this footnote marker needs a space; GitHub at least doesn't render it: 
<img width="428" alt="image" src="https://user-images.githubusercontent.com/17170027/157748340-3cd60e25-514e-4d5e-b77a-29f018780251.png">

Reply:
Fixed.

Reply:
I still see markup here, rather than a valid footnote link, when I view the rendered document.

Reply:
Are you on the latest version? You're commenting on an outdated diff. This has been sort of fixed (the spacing issue) on the latest commit. I don't see it having a matching footnote, however.